plugin/pkg/admission/security/podsecuritypolicy/admission_test.go

```diff
@@ -251,6 +252,15 @@ func TestAdmitPrivileged(t *testing.T) {
 	}
 }
 
+func TestAdmitMirrorPod(t *testing.T) {
+	nonMirrorPod := goodPod()
+	testPSPAdmit("non-mirror pod", nil, nonMirrorPod, false, "", t)
+
+	mirrorPod := goodPod()
+	mirrorPod.Annotations[kapi.MirrorPodAnnotationKey] = "true"
```

Comment:
As far I understand, the annotation can have any values. I'd suggest to put a different value here, otherwise, it could be interpreted as "the annotation should have "true" as a value".